third-party/qthread/Makefile

```diff
@@ -120,17 +107,13 @@ endif
 HAVE_GUARD_PAGES = 0
 ifeq (, $(call isTrue, $(CHPL_QTHREAD_NO_GUARD_PAGES)))
 HAVE_GUARD_PAGES = 1
-CHPL_QTHREAD_CFG_OPTIONS += --enable-guard-pages
+CHPL_QTHREAD_CFG_OPTIONS += -DQTHREADS_GUARD_PAGES=ON
 endif
 
-# optionally enable printing compilation commands, etc. (to help diagnose
-# build problems)
 ifneq (, $(CHPL_QTHREAD_LOUD_RULES))
-CHPL_QTHREAD_CFG_OPTIONS += --disable-silent-rules
+CHPL_QTHREAD_CFG_OPTIONS += -DCMAKE_VERBOSE_MAKEFILE=ON
 endif
 
-CHPL_QTHREAD_CFG_OPTIONS += --enable-static --disable-shared
-
```

Comment:
why is this removed? is it now the default?

Reply:
This is covered later by setting `BUILD_SHARED_LIBS=OFF`.

Reply:
Ah I missed that, thanks!